publication-model/src/test/java/no/unit/nva/model/PublicationJournalArticleTest.java

```diff
@@ -39,14 +38,14 @@ public static Stream<Class<?>> journalArticleInstanceTypes() {
     @Test
     void objectMappingOfPublicationClassReturnsSerializedJsonWithJsonLdFrame() throws IOException {
 
-        Publication publication = PublicationGenerator.randomPublication();
+        var publication = PublicationGenerator.randomPublication();
         var title = publication.getEntityDescription().getMainTitle();
 
-        JsonNode publicationWithContext = toPublicationWithContext(publication);
+        var publicationWithContext = toPublicationWithContext(publication);
 
-        Object framedPublication = produceFramedPublication(publicationWithContext);
+        var framedPublication = produceFramedPublication(publicationWithContext).toString();
 
-        Assertions.assertTrue(JsonUtils.toString(framedPublication).contains(title));
+        Assertions.assertTrue(framedPublication.contains(title));
```

Comment:
Honestly, I don't see the point in this test at all — we should review testing in model.

Reply:
Agree